docs/pages/guides/authentication/firebase.mdx

```diff
@@ -57,6 +58,22 @@ exports.auth = functions.https.onCall((data, context) => {
 });
 ```
 
+The `userId` used in the `authorize` function corresponds to userId used in our APIs when setting permissions (e.g. in create room).
+You can use the following example to use data from `userId` and `userInfo on the front end.
+```ts
+function Component() {
+  const self = useSelf();
+
+  // "123"
+  console.log(self.id);
+
+  // "red"
+  console.log(self.info.color);
+
+  // ...
+}
```

Comment:
```suggestion
const self = room.getSelf();

// "123"
console.log(self.id);

// "red"
console.log(self.info.color);
```

Firebase could be any framework, so using `@liveblocks/client` instead of `@liveblocks/react` here